protoc-gen-twirp/generator.go

```diff
@@ -61,14 +64,43 @@ func newGenerator() *twirp {
 	t := &twirp{
 		pkgs:                make(map[string]string),
 		pkgNamesInUse:       make(map[string]bool),
+		importMap:           make(map[string]string),
 		fileToGoPackageName: make(map[*descriptor.FileDescriptorProto]string),
 		output:              bytes.NewBuffer(nil),
 	}
 
 	return t
 }
 
+// commandLineParameters breaks the comma-separated list of key=value pairs
+// in the parameter (a member of the request protobuf) into a key/value map.
+// It then sets file name mappings defined by those entries.
+func (t *twirp) commandLineParameters(parameter string) {
```

Comment:
Instead of making this a method on `*twirp`, could you make this a function at the top level that returns a struct that has the fields that come from the command line? I think that would make it easier to write unit tests and also make it easier to document and keep track of which parameters are set in the command line like this. 

I'm picturing the call site as something like this: 
```
params, err := parseParams(in.GetParameters())
if err != nil {
  gen.Fail("invalid parameters passed to --twirp_out: %v", err)
}
t.importMap = params.importMap
t.importPrefix = params.importPrefix
```